src/trajectories/transition.jl

```diff
@@ -110,55 +121,52 @@ See also [`transition`](@ref).
 
 > An example script using `transitions` is available [here](https://github.com/juliadynamics/CriticalTransitions.jl/blob/main/examples/sample_transitions_h5.jl).
 """
+
 function transitions(
     sys::CoupledSDEs,
-    x_i,
-    x_f,
-    N=1;
-    rad_i=0.1,
-    rad_f=0.1,
+    x::NTuple{2},
```

Comment:
What is the reason for switching from `x_i`, `x_f` to a tuple `x`? I find the former much more intuitive since there is a direction associated with the transition

Reply:
Ha yeah, I liked it because it was more concise. I am not a fan of too many `args` and `kwargs` in function. But I don't have strong opinion, if you want it reversed, I'll do it :)

Reply:
Are you okay with adding both methods?

Reply:
I agree that we shouldn't overload functions but in this case I prefer having the arguments separately as I find it more clear. Having both methods seems a bit overkill if the only difference is the input as tuple vs. two args?